scripts/crystals.js

```diff
@@ -1574,5 +1574,41 @@ BDO.crystals = [
+        "name": "Ah'krad Crystal",
+        "grade": "4",
+        "socket": "Awakening Weapon",
+        "effect": "All Accuracy +3, Extra AP against Monsters +5",
```

Comment:
I know the DB has it lowercase on one but uppercase on the other, but can you capitalize "against" here, so that the drop down doesn't have two separate entries? Thanks for taking the time to make a PR. :)

Reply:
Done